statistics/table.go

```diff
@@ -113,51 +136,56 @@ func (h *Handle) TableStatsFromStorage(ctx context.Context, info *model.TableInf
 	for _, row := range rows {
 		distinct := row.Data[3].GetInt64()
 		histID := row.Data[2].GetInt64()
+		histVer := row.Data[4].GetUint64()
 		if row.Data[1].GetInt64() > 0 {
 			// process index
-			var idx *Index
-			for _, idxInfo := range info.Indices {
+			idx := table.Indices[histID]
+			for _, idxInfo := range tableInfo.Indices {
```

Comment:
This part of code will refactor after we can process the psuedo column correctly.